src/c/types/sequence.c

```diff
@@ -230,6 +272,12 @@ bool deserialize_endian_sequence_uint64_t(MicroBuffer* buffer, const Endianness
     return deserialize_endian_array_uint64_t(buffer, endianness, array, *size);
 }
 
+bool deserialize_endian_sequence_int8_t(MicroBuffer* buffer, const Endianness endianness, int8_t* array, uint32_t* size)
+{
+    deserialize_endian_uint32_t(buffer, endianness, size);
```

Comment:
This function is not safe. First call can fail but second not. Example: Array of one element when buffer is only 3 bytes left.

Reply:
Done!